pages/establishment/pdf/index.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import fetch from 'r2';
+import { Router } from 'express';
+import { renderToStaticMarkup } from 'react-dom/server';
+import Body from './views';
+import Header from '../../common/views/pdf/header';
+import Footer from '../../common/views/pdf/footer';
+import content from './content';
+
+const namedPeople = establishment => {
+  const profiles = {};
+
+  ['pelh', 'holc', 'nprc', 'nio', 'ntco', 'nvs', 'nacwo'].forEach(roleType => {
+    profiles[roleType] = establishment.roles.filter(role => role.type === roleType).map(role => role.profile);
+  });
```

Comment:
https://lodash.com/docs/4.17.15#groupBy

Reply:
Only problem with `groupBy` here is that it won't populate role types that aren't assigned, so I'll have to check for their existence when I use em.

Reply:
Is that ok? It's possibly desirable that some roles - specifically NPRC - don't appear if there are none.

As an aside, `holc` probably shouldn't be in this list as it's not a licensed role.

Reply:
Hold doesn’t get used but I’ll remove it

Reply:
I see HOLC isn't rendered anyway. We should remove it from this list.

Reply:
Please can you add a suggestion with it removed and I’ll commit it via phone 🙂